bayesian/utils.py

```diff
@@ -624,15 +627,17 @@ def _check_usage_outlier(self, pkg_name, manifest_response):
 
     def _companion_reason(self, manifest_response):
         """
-        It will populate the simple reason for each companion package
+        It will populate the simple reason for each companion package.
         :param manifest_response: dict. object having all recommendation elements
         :return: same dict. object with populated reasons for each companion package
         """
         for pkg in manifest_response[0].get("recommendation", {}).get("companion", []):
             name = pkg.get("name")
-            stack_count = str(pkg.get("cooccurrence_probability"))
-            sentence = "Package {} appears in {} different stacks along with the provided input " \
-                       "stack. Do you want to consider adding this Package?"\
-                .format(name, stack_count)
-            pkg["reason"] = sentence
+            stack_confidence = str(pkg.get("cooccurrence_probability"))
+            stack_count = str(pkg.get("cooccurrence_count"))
+            count_sentence = "Package {} appears in {} different stacks".format(
+                name, stack_count)
```

Comment:
I would not add the whole sentences into the REST API response. This is job for a consumer to do it properly (with possible translations etc. etc.)

Reply:
@tisnik - IMHO, client should be dumb enough to populate key value pairs. Having said that, we maintain different UI (OSIO stack report, stack-report portal, VSCode IDE, Devstudio IDE) and in the future we will add more. Hence it made more sense to send the reasoning from API server so that all clients gets exact messaging as we need plus lot less maintenance. I hope I answered your query.

Reply:
@miteshvp OK, so @sara-02  would you kindly update documentation of our API endpoint to reflect this new field?

Reply:
@pkajaba please share the link for the same.

Reply:
cc @samuzzal-choudhury I can't find it, but there should be anything like that, right?

Reply:
@pkajaba does not look like it is in this repo, will have to create a new PR in which ever repo it is.